charts/backingservices/charts/constellation-messaging/templates/_messaging-registry-secret.tpl

```diff
@@ -0,0 +1,14 @@
+{{- define "messagingRegistryCredentialsSecretTemplate" }}

```

Comment:
@alemax22 , Could you include this for messaging charts like  charts/backingservices/charts/constellation/templates/clln-registry-secret.yaml ?